deploy/olm-catalog/ibm-mongodb-operator/1.1.5/ibm-mongodb-operator.v1.1.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,336 @@
+                    command:
+                    - sh
+                    - -c
+                    - exec echo echo start mongodb-operator
+                  initialDelaySeconds: 10
+                  timeoutSeconds: 2
+                resources:
+                  limits:
+                    cpu: 40m
+                    memory: 200Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+                  capabilities:
+                    drop:
+                      - ALL
+                  privileged: false
+                  readOnlyRootFilesystem: true
+                  runAsNonRoot: true
+              serviceAccountName: ibm-mongodb-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - services/finalizers
+          - serviceaccounts
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - ibm-mongodb-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - operator.ibm.com
+          resources:
+          - mongodbs
+          - mongodbs/finalizers
+          - mongodbs/status
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - certmanager.k8s.io
+          resources:
+          - certificates
+          - certificaterequests
+          - orders
+          - challenges
+          - issuers
+          verbs:
+          - '*'
+        serviceAccountName: ibm-mongodb-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - IBM
+  - Cloud
+  labels:
+    name: ibm-mongodb-operator
+  links:
+  - name: IBM MongoDB Operator Project
+    url: https://github.com/IBM/ibm-mongodb-operator
+  maintainers:
+  - email: supports@ibm.com
+    name: IBM Support
+  maturity: alpha
+  provider:
+    name: IBM
+  replaces: ibm-mongodb-operator.v1.0.0
```

Comment:
The replaces version should be 1.1.4

Reply:
I found the previous CSVs all set the  `replaces: ibm-mongodb-operator.v1.0.0`. Seems it doesn't matter when the skip range is set correctly.